m3/README.md

```diff
@@ -0,0 +1,8 @@
+# JMH Benchmark Tests results
+
+Benchmark                                                     Mode  Cnt    Score    Error   Units
```

Comment:
Btw @SokolAndrey do we have a sense of what the allocation profiles look like here? cc. @alexeykudinkin for approaches on getting that (I think Andrei mentioned utilizing the IntelliJ profiler there, which might work).

Reply:
No need for external profilers, you can use JMH internal profilers for that.

https://stackoverflow.com/a/22642703

Reply:
Ooh nice that's much better, thanks!

Reply:
added